SOURCES/xapi-nbd-1.11.0-adapt-vbd.create-call.XCP-ng.patch

```diff
@@ -0,0 +1,22 @@
+Adapt `VBD.create` call to new API.
+See: https://github.com/xapi-project/xen-api/commit/07b0fd4d509825277d44258da29de37d1d758aff
+
+commit a21abbd
+Author: BenjiReis <benjamin.reis@vates.fr>
+Date:   Mon Jul 6 10:12:18 2020 +0200
+
+    toto
```

Comment:
Toto?

Reply:
Yeah, it was a dummy commit so I just removed it. The explain text is above.